### PR TITLE
pricing card is now selectable on click

### PR DESCRIPTION
**Pull request by geclos**
Pricing card is now selectable with custom color on click.

screen shot 2017-06-26 at 12 41 39 pm

### CODE REVIEW

File: src/components/Steps/index.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+              style={{ backgroundColor: lineActive ? color : '#DFE3E6' }}
+              className={cx('line', { active: lineActive })}
+            />
+          </div>
+        )
+      })}
+    </div>
+  )
+}
+
```
Comment (andresgutgon) [inline src/components/Steps/index.jsx:42]: Why is this file here?

Comment (geclos) [reply src/components/Steps/index.jsx:42]: I needed both to be in the same place to build the library and work with it in `factorial-public`. 

I'll remove the commit before merging.
